website/src/models/group.js

```diff
@@ -141,7 +141,8 @@ var cleanQuestProgress = function(merge){
       down: 0,
       collect: {}
     },
-    completed: null
+    completed: null,
+    invited: false
```

Comment:
Yes. It's used [here](https://github.com/HabitRPG/habitrpg/blob/0fa93506f5ea2eb68284486dc0a37bd970c1e7a3/website/src/controllers/groups.js#L835) when the quest is started, and takes care of any member who didn't accept or reject the quest.